UGW: fix to avoid compiling toolchain when external toolchain is set

diff --git a/Makefile b/Makefile
--- a/Makefile
+++ b/Makefile
@@ -36,7 +36,9 @@ else
   include target/Makefile
   include package/Makefile
   include tools/Makefile
+ifeq ($(CONFIG_EXTERNAL_TOOLCHAIN),)
   include toolchain/Makefile
+endif
 
 $(toolchain/stamp-install): $(tools/stamp-install)
 $(target/stamp-compile): $(toolchain/stamp-install) $(tools/stamp-install) $(BUILD_DIR)/.prepared
